lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -31,8 +31,37 @@ def colorized_message_lines(colorizer=::RSpec::Core::Formatters::ConsoleCodes)
           end
         end
 
-        def formatted_backtrace
-          backtrace_formatter.format_backtrace(exception_backtrace, example.metadata)
+        def formatted_backtrace(exception=@exception)
+          lines = backtrace_formatter.format_backtrace((exception.backtrace || []), example.metadata)
+
+          lines + formatted_cause(exception)
```

Comment:
I think it makes sense to collapse these lines and just break it up over two lines. e.g.

```Ruby
backtrace_formatter.format_backtrace((exception.backtrace || []), example.metadata) +
  formatted_cause(exception)
```